op-node/p2p/sync.go

```diff
@@ -670,20 +668,6 @@ func (s *SyncClient) readExecutionPayload(data []byte, expectedTime uint64) (*et
 		return nil, fmt.Errorf("failed to decode response: %w", err)
 	}
 
-	// [Kroma: START] Use KromaDepositTx instead of DepositTx before MPT time
-	if !s.cfg.IsKromaMPT(expectedTime) {
-		for i, otx := range res.Transactions {
-			if otx[0] == types.DepositTxType {
-				var err error
-				res.Transactions[i], err = derive.ToKromaDepositBytes(otx)
-				if err != nil {
-					return nil, fmt.Errorf("failed to convert deposit tx to KromaDepositTx: %w", err)
-				}
-			}
-		}
-	}
-	// [Kroma: END]
-
```

Comment:
Curious, why is this part removed?

Reply:
This part handles unsafe payload received via p2p. 
Since the payload delivered by peers will already include deposit txs converted to KromaDepositTx, this conversion in this part has been removed.